static/js/formatters-internal.js

```diff
@@ -294,7 +294,7 @@ export function image(simpleOrComplexImage = {}, size = '200x', atLeastAsLarge =
     return img;
   }
 
-  function imageBySizeEntity(image, desiredSize, atLeastAsLarge = true) {
+  function createDynamicUrl(image, desiredSize, atLeastAsLarge = true) {
```

Comment:
I don't know if it needs to its own separate function if it's call once here only and it contains almost all of function `image`'s logic. Maybe we could just use the logic as is without the function or move this out to its own function instead of nested inside image?

Reply:
updated!